common/schema/schema.go

```diff
@@ -255,9 +255,13 @@ func (s *Schema) InternalModules() []*Module {
 	return out
 }
 
-func (s *Schema) IncludeBuiltins() *Schema {
+// WithBuiltins returns a new schema with the builtins module added to each internal realm.
+func (s *Schema) WithBuiltins() *Schema {
+	c := new(Schema)
+	*c = *s
```

Comment:
This is not going to be an actual copy right? You'll have to use `internal/reflect.DeepCopy()` if that's your intent here.

Reply:
No, this is a shallow copy on purpose. No need for deep copy here.